ExomeDepth/settings.py

```diff
@@ -18,7 +18,9 @@
 reference_genome = "/hpc/diaggen/data/databases/ref_genomes/Homo_sapiens.GRCh37.GATK.illumina/Homo_sapiens.GRCh37.GATK.illumina.fasta"
 
 #Used R version
-r_version = "R/3.5.1"
+r_library_path = "/data/software/tools/R_libs/3.5.1"
```

Comment:
Uiteindelijk als default locatie wel hpc path hier neerzetten.
/hpc/diaggen/software/production/R_libs/ExomeDepthv2.0.0/3.5.1_singularity/

Reply:
Done in 6c03a2a